app/controllers/provider/admin/account/email_configurations_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Provider::Admin::Account::EmailConfigurationsController < Provider::Admin::Account::BaseController
+
+  # We only enable it for master for now, feel free to roll it out for any provider
+  before_action :ensure_master_domain, :email_configurations_enabled?
+
+  def index
+  end
+
+  def new
+    @email_configuration = EmailConfiguration.new
+  end
+
+  def edit
+    @email_configuration = account.email_configurations.find(params[:id])
+  end
+
+  def update
+    @email_configuration = account.email_configurations.find(params[:id])
+
+    if @email_configuration.update(configuration_params)
+      redirect_to action: :index
```

Comment:
Some flash messages would be nice I guess: _Email configuration created_, _Email configuration could not be updated_, etc.